pyocd/probe/pydapaccess/dap_access_cmsis_dap.py

```diff
@@ -593,14 +609,14 @@ def unlock(self):
 
     def _read_protocol_version(self):
         """! Determine the CMSIS-DAP protocol version."""
-        fw_version_str = self._protocol.dap_info(self.ID.FW_VER)
+        protocol_version_str = self._protocol.dap_info(self.ID.CMSIS_DAP_PROTOCOL_VERSION)
         # Just in case we don't get a valid response, default to the lowest version (not including betas).
-        if not fw_version_str:
+        if not protocol_version_str:
             self._cmsis_dap_version = CMSISDAPVersion.V1_0_0
         # Deal with DAPLink broken version number, where these versions of the firmware reported the DAPLink
         # version number for DAP_INFO_FW_VER instead of the CMSIS-DAP version, due to a misunderstanding
         # based on unclear documentation.
-        elif (self._vidpid == ARM_DAPLINK_ID) and (fw_version_str in ("0254", "0255", "0256")):
+        elif (self._vidpid == ARM_DAPLINK_ID) and (protocol_version_str in ("0254", "0255")):
```

Comment:
Is the removal of `"0256"` related to this new protocol version?

Reply:
It's because DAPLink version 0256 hasn't been released yet, and the incorrect reporting of the DAPLink version number instead of the protocol version has already been fixed.